app/frontend/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,42 @@
+
+.navbar {
+  margin-left: 300px;
+  display: flex;
+  // justify-content: space-around;
+  align-items: center;
+
+      .logo {
+        margin-right: 120px;
+      }
+
+      ul {
+        margin: auto;
+        display: flex;
+        justify-content: space-around;
+        max-width: 400px;
+      }
+
+      ul li {
+        list-style: none;
+        padding: 40px ;
+        position: relative;
+        bottom: 10px;
+      }
+
+      li:not(:first-child)::before {
+        content: "*";
+        position: absolute;
+        left: -5px;
+        top: 46px;
+        color: #fff;
+      }
+
+      ul li a {
+        margin-left: 20px;
+        text-decoration: none;
+        color: #fff;
+        position: relative;
+        font-size: 20px;
+        font-size: 18px;
+      }
```

Comment:
The indentation isn't right.

Also please use SCSS' nesting like so
```
ul {
  ...
  
  li {
    ...
    
    &:not(:first-child)::before {
      ...
    }
    
    a {
      ...
    }
  }
}
```

Instead of using margin-right on the .logo use gap on the .navbar

- You can remove the max-width and the margin auto from the ul
- The list-style of the li is already none from the config/_setup.scss file
- give the padding of the li to the ul instead and give the ul a gap to create spacing between the li
- Do not use bottom on the li
- ::before will be difficult to be responsive. Just add <li>*</li> between the links you already have in the html that'll be easier to manage
- don't use margin or position on the a tags
- text-decoration is already set in the config/_setup so you can remove it
- you have 2 font-sizes in the a tag, remove one